__tests__/server.test.js

```diff
@@ -111,3 +111,61 @@ describe("GET /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id/comments", () => {
+  test("should return 200 on successful call", () => {
+    return request(app).get("/api/articles/1/comments").expect(200);
+  });
+
+  test("should return a list of all the comments attached to that article id", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.comments.length).toBe(11);
+        body.comments.forEach((comment) => {
+          expect(comment).toHaveProperty("comment_id");
+          expect(comment).toHaveProperty("votes");
+          expect(comment).toHaveProperty("created_at");
+          expect(comment).toHaveProperty("author");
+          expect(comment).toHaveProperty("body");
+          expect(comment).toHaveProperty("article_id");
+        });
+      });
+  });
+
+  test('should return a 400 error with the message "Invalid data type" when not provided a number', () => {
+    return request(app)
+      .get("/api/articles/one/comments")
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.message).toBe("Invalid data type");
+      });
+  });
+
+  test('should return a 404 error with the message "Article not found" when provided an incorrect id', () => {
+    return request(app)
+      .get("/api/articles/70650/comments")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.message).toBe("Article not found");
+      });
+  });
+
+  xtest("should return a 404 with 'No comments available' if there are no comments attached to the article", () => {
```

Comment:
This test won't pass currently. Remove the x and implement the behaviour needed to make it pass 👍 !